Blockzilla/URLBar.swift

```diff
@@ -601,6 +632,10 @@ class URLBar: UIView {
 
         delegate?.urlBarDidDeactivate(self)
     }
+    
+    @objc private func didPressPageActions() {
```

Comment:
Is this going to be handled in a separate PR or was it meant to be filled in?

Reply:
Will be handled with https://github.com/mozilla-mobile/focus-ios/issues/1060.